lib/ael_api/application.ex

```diff
@@ -17,14 +17,18 @@ defmodule Ael do
     children = [
       # Start the endpoint when the application starts
       supervisor(Ael.Web.Endpoint, []),
+      supervisor(Registry, [:unique, Ael.Registry]),
       # Starts a worker by calling: Ael.Worker.start_link(arg1, arg2, arg3)
       # worker(Ael.Worker, [arg1, arg2, arg3]),
     ]
 
     # See http://elixir-lang.org/docs/stable/elixir/Supervisor.html
     # for other strategies and supported options
     opts = [strategy: :one_for_one, name: Ael.Supervisor]
-    Supervisor.start_link(children, opts)
+
+    children
+    |> Supervisor.start_link(opts)
+    |> register_gcs_config()
```

Comment:
You don't need to pipe application trough this function. Simply save result and call function after it.